vm/chef/cookbooks/singlefs/files/singlefs-monitoring

```diff
@@ -0,0 +1,427 @@
+#!/bin/bash -eu
+#
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to configure local monitoring for the single node file server.
+
+set -e
+set -x
+
+. /opt/c2d/get_metadata.sh
+. /opt/c2d/instance_utils.sh
+
+if (( "$(id -u)" > 0 )); then
+  echo "Please run this script as root."
+  exit 1
+fi
+
+# Only configure monitoring once.
+readonly COMPLETE_FILE=/opt/c2d/monitoring.complete
+if [[ -f "${COMPLETE_FILE}" ]]; then
+  echo "Monitoring configuration already completed."
+  exit 0
+fi
+touch "${COMPLETE_FILE}"
+
+# Redirect all output to the log file.
+readonly LOG_FILE=/opt/c2d/monitoring.log
+exec > >(tee -a "${LOG_FILE}" ) 2>&1
+
+readonly DISABLE_MONITORING="$(get_metadata instance/attributes/DISABLE_MONITORING || true)"
```

Comment:
**(Postpone action):** This attribute doesn't exist in metadata. Should be removed?